### PR TITLE
bucketName as a global secondary index

### PR DESCRIPTION
**Pull request by amirbey**

### CODE REVIEW

**Review by davemcorwin — APPROVED** [review]
Looks 💯 ! You'll have to teach me all the things when we get a chance!

**Comment by amirbey** [comment]
:eyes @davemcorwin  - requesting re-review ... i made a minor change ... https://github.com/18F/federalist-proxy-edge/pull/22/commits/56fc63555765591dcaff7c7b43c7138f9395104b

**Review by davemcorwin — APPROVED** [review]

**Comment by amirbey** [comment]
@davemcorwin - sorry, a few more changes ... not expecting any further change son this PR

File: template.yaml [hunk 1]
```diff
@@ -115,14 +115,25 @@ Resources:
...
     Properties:
       TableName: !Sub "federalist-proxy-${StackEnv}"
-      PrimaryKey:
-        Name: Id
-        Type: String
-      Tags:
-          AppType: Serverless
+      AttributeDefinitions:
+        - AttributeName: Id
+          AttributeType: S
```
Comment (davemcorwin) [inline template.yaml:123]: the type is just `S`?

**Review by davemcorwin — APPROVED** [review]

File: template.yaml (hunk 1 above)
Comment (amirbey) [reply template.yaml:123]: yep .. the options are :
S -> String
N -> Number
B -> Binary

**Pull request merged**